crates/hc_sandbox/tests/cli.rs

```diff
@@ -225,6 +249,7 @@ fn get_sandbox_command() -> Command {
 async fn get_launch_info(stdout: &mut ChildStdout) -> LaunchInfo {
     let mut lines = BufReader::new(stdout).lines();
     while let Ok(Some(line)) = lines.next_line().await {
+        println!("@@@@@-{line}-@@@@@");
```

Comment:
Is this just stylistic, or actual magic? Is there some code somewhere else that's supposed to parse this pattern? If so comment please.

Reply:
Just stylistic. I needed to visually see what it was outputting so I could figure out where it wasn't working while debugging the test.